xrpl_api/src/types/amount.rs

```diff
@@ -39,13 +39,6 @@ impl Amount {
             Currency::Xrp => Self::xrp(value),
         }
     }
-
-    pub fn size(&self) -> f64 {
```

Comment:
Seems this was returning the "value" of the `Amount`. I removed it because it did not seem production ready (using unwrap e.g.), so I think it is better not to have it present.

Reply:
In future PRs (esp. in large ones) maybe it's better to just comment the code and add the rationale for the depreciation. But it's OK for this PR.

Reply:
Yeah you're right, it could also just have been deprecated